docs/docsite/rst/user_guide/playbooks_filters_ipaddr.rst

```diff
@@ -405,6 +405,32 @@ be automatically converted to a router address (with a ``::1/48`` host address):
 
 .. _6to4: https://en.wikipedia.org/wiki/6to4
 
+Getting additional information from network address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
This section would be stronger with an over-arching description of what these filters do. I'm guessing a bit on the wording, but something like this:

```suggestion

To find usable IP addresses within an IP range, try these ``ipaddr`` filters:

```